contents/terms.md

```diff
@@ -3,7 +3,84 @@ title: Terms
+
+## 1. License and support
+1.1 Subject to the terms and conditions of this Agreement, PostHog hereby grants to Customer and its Affiliates (as defined below) a limited, non-exclusive, non-transferable, non-sublicensable license for Customer’s and its Affiliates’ employees and contractors to (1) internally (a) use, reproduce, modify, prepare derivative works based upon, and display the code of PostHog Cloud Edition at the tier level selected by Customer (or set forth on a Quote (as defined below), if applicable with the specifications generally promulgated by PostHog from time to time (the “Software”), solely (i) for its internal use in connection with the development of Customer’s and/or its Affiliates’ own software, and (ii) at the level of usage for which Customer has paid PostHog; and (b) use the documentation, training materials or other materials supplied by PostHog (the “Other PostHog Materials”); and (2) modify the Software and publish patches to the Software, solely at the level of usage for which Customer has paid PostHog. Notwithstanding anything to the contrary, Customer agrees that PostHog and/or its licensors (as applicable) retain all right, title and interest in and to all Software incorporated in such modifications and/or patches, and all such Software may only be used, copied, modified, displayed, distributed, or otherwise exploited in full compliance with this Agreement, and with a valid PostHog Cloud Edition subscription for the correct level of usage. The Software and Other PostHog Materials are collectively referred to herein as the “Licensed Materials.” “Affiliate” means any entity(ies) controlling, controlled by, and/or under common control with a party hereto, where “control” means the ownership of more than 50% of the voting securities in such entity. “User” means each individual end-user (person or machine) of Customer and/or its Affiliates (including, without limitation, employees, agents or consultants thereof) with access to the Licensed Materials hereunder.
+
+1.2 Subject to the terms hereof, PostHog will provide reasonable support to Customer for the Licensed Materials as set forth on the 'Features' page, for the support plan selected and paid for by Customer. Notwithstanding anything to the contrary, in the event that Customer does not reasonably comply with written specifications or instructions from PostHog’s service engineers regarding any support issue or request (including without limitation, failure to make backups of Customer’s Licensed Materials) (each, a “Support Issue”), PostHog may terminate its support obligations to Customer with respect to such Support Issue upon fifteen (15) days’ written notice if Customer does not cure such noncompliance within the notice period.
+
+1.2.1 PostHog will use reasonable commercial efforts to respond to support questions by phone or email during the next business day at the latest. The number of support questions is not limited.
```

Comment:
I don't think we can offer support like this for a $0.01 a month user when we tweak to growth based pricing, perhaps remove the timeline here.

Reply:
Agreed - updated